tracer/build/_build/Honeypot/DependabotFileManager.cs

```diff
@@ -54,7 +54,14 @@ public static Task UpdateIntegrations(AbsolutePath honeypotProject, List<Integra
                     fakeRefs += $@"{Environment.NewLine}    <!--    Assembly: {integration.AssemblyName} -->";
                 }
 
-                fakeRefs += $@"{Environment.NewLine}    <!-- Latest package https://www.nuget.org/packages/{package.NugetName}/{package.LatestVersion} -->";
+                // HACK: Add callout for versions that aren't tested - ideally we find a better way to do this
+                var untestedCallout = "";
+                if (package.LatestTestedVersion is not null && package.LatestVersion > package.LatestTestedVersion)
+                {
+                    untestedCallout = $"\u26A0POTENTIALLY UNSUPPORTED\u26A0 ";
```

Comment:
⚠POTENTIALLY UNSUPPORTED⚠ 

is the Unicode

Reply:
not saying you should change this, but just fyi you can use the "⚠" character directly in the string

```suggestion
                    untestedCallout = $"⚠POTENTIALLY UNSUPPORTED⚠ ";
```

Reply:
should we add some spaces, though?
```suggestion
                    untestedCallout = $"\u26A0 POTENTIALLY UNSUPPORTED \u26A0 ";
```